app/controlplane/cmd/main.go

```diff
@@ -218,3 +224,16 @@ func initSentry(c *conf.Bootstrap, logger log.Logger) (cleanupFunc func(), err e
 func newProtoValidator() (*protovalidate.Validator, error) {
 	return protovalidate.New()
 }
+
+func newSigningCA(_ context.Context, c *conf.Bootstrap, logger log.Logger) (ca.CertificateAuthority, error) {
```

Comment:
instead of conf.Bootstrap it might make sense to have `CertificateAuthority` instead, that way you simplify potential testing.